test/jdk/java/awt/Headless/HeadlessMalfunctionAgent.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2024, 2025, Oracle and/or its affiliates. All rights reserved.
```
The first year should be preserved; when there are two years, the last one gets updated.